### DISCUSSION TITLE
Lens ignores perPageOptions

### DISCUSSION THREAD

**Discussion by puzzledmonkey** [laravel/nova-issues#5764] [Ideas & Feature Requests]
- Laravel Version: 4.26.5
- Nova Version: 10.16.1
- PHP Version: 8.2
- Database Driver & Version: n/a
- Operating System and Version: n/a
- Browser type and version: n/a
- Reproduction Repository: n/a

### Description:

Lenses ignore perPageOptions and fall back to the resource per page options

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. Make a Lens
2. Add `public static $perPageOptions = [50, 100];` to the Lens class
3. Observe the Lens uses the resource perPageOptions (defaults to 25/50/100)

I'm not sure if this counts as a new feature but it would make sense to me if the lens per page options could be set independently of the main resource's.

**Comment by crynobone** [comment]
Lens doesn't have that options and only reuse the resource perPageOptions

**Reply by puzzledmonkey** [reply]
it would be really useful if we could set this for lenses (separately from the resources themselves), and it doesn't seem like it would be a tricky thing to add with any luck..?

**Comment by puzzledmonkey** [comment]
Closing due to apathy
